gazebo_plugins/test/worlds/wheelslip_worlds/gazebo_ros_wheelslip_2.world

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" ?>
+<!--
+  Gazebo ROS wheel slip plugin demo
+  Try for example:
+  To change slip compliance of a group of wheels:
+    ros2 param set /trisphere_cycle_slip0/wheel_slip_rear slip_compliance_unitless_longitudinal 1.0
+  To get the current slip compliance of a group of wheels:
+    ros2 param get /trisphere_cycle_slip0/wheel_slip_rear slip_compliance_unitless_longitudinal
+-->
+<sdf version="1.6">
+  <world name="default">
+    <gravity>-2 0 -9.8</gravity>
+
+    <include>
+      <uri>model://ground_plane</uri>
+    </include>
+    <include>
+      <uri>model://sun</uri>
+    </include>
+
+    <include>
+      <uri>model://trisphere_cycle</uri>
+      <name>trisphere_cycle_slip0</name>
+      <pose>0 0 0  0 0 0</pose>
+      <plugin name="wheel_slip_front" filename="libgazebo_ros_wheel_slip.so">
+
+        <ros>
+          <namespace>trisphere_cycle_slip0</namespace>
+          <parameter name="slip_compliance_unitless_lateral" type="double">10.0</parameter>
+          <parameter name="slip_compliance_unitless_longitudinal" type="double">11.0</parameter>
+        </ros>
+
+        <wheel link_name="wheel_front">
+          <slip_compliance_lateral>1</slip_compliance_lateral>
+          <slip_compliance_longitudinal>2</slip_compliance_longitudinal>
+          <wheel_normal_force>77</wheel_normal_force>
+        </wheel>
+      </plugin>
+
+      <plugin name="wheel_slip_rear" filename="libgazebo_ros_wheel_slip.so">
+        
+        <ros>
+          <namespace>trisphere_cycle_slip0</namespace>
+        </ros>
+       
+       <wheel link_name="wheel_rear_left">
+          <slip_compliance_lateral>-3</slip_compliance_lateral>
+          <slip_compliance_longitudinal>4</slip_compliance_longitudinal>
+          <wheel_normal_force>32</wheel_normal_force>
+        </wheel>
+        <wheel link_name="wheel_rear_right">
+          <slip_compliance_lateral>5</slip_compliance_lateral>
+          <slip_compliance_longitudinal>6</slip_compliance_longitudinal>
+          <wheel_normal_force>32</wheel_normal_force>
+        </wheel>
+      </plugin>
+      <plugin name="joint_control" filename="libJointControlPlugin.so">
```

Comment:
Is this joint controller necessary for the tests or can we remove it from the world file?

Reply:
Removed